Automattic/wp-calypso
# Login: Show user and password form

Related to https://github.com/Automattic/wp-calypso/issues/95453
Fixes https://github.com/Automattic/wp-calypso/issues/95453

## Proposed Changes

* Show the login form when navigating back from the “Email me a login link” page using the `enter a password instead` link button.

## Why are these changes being made?


* Currently there's a loop where users who have a last login method 'saved' and navigate back from the “Email me a login link” page. Check video for better understanding
* This PR shows the login form when using the link button, upon refresh, users should see the last used login method.
U01CRQSPUUX/F07RRQ9AA8M-slack-file

## Testing Instructions



* Use live link or pull this branch
* Log in using any method, then logout
* Navigate to log in again and see the “previously used method”
* Click on “Email me a login link” and then “Enter a password instead”
* When back on login, you should see the login for with username field empty
* Upon refresh of the login page, you should see the “previously used method” again


## Pre-merge Checklist
